### ISSUE TITLE
vyos: oxidized can't get config when logged in as non-local, non-privileged user

### ISSUE THREAD

**Issue by h-lopez** [ytti/oxidized#3111]
VyOS allows you to configure TACACS+/RADIUS authentication in addition to local accounts. If oxidized is using a TACACS-backed account with a privilege-level of 1 it fails to pull the configuration of the remote device.

enabling debug shows oxidized manages to successfully login, but shows no further output after oxidized gets the shell prompt.
```
Last login: Mon Mar 25 16:47:38 2024 from 172.21.50.56

oxidized@er1-devlab1> 
```

this appears to be due to the prompt-matching oxidized does on this model only works for privileged accounts.
by default oxidized matches a prompt in the format:
`oxidized@er1-devlab1:~$` 

however TACACS-backed accounts with limited privileges on VyOS will instead see a prompt in the format:
`oxidized@er1-devlab1>`

**Comment by h-lopez** [comment]
yes, operator level accounts still exist if you're using a TACACS or RADIUS backed account on 1.3, 1.4epa and the recent 1.5 rolling releases.

I can confirm this is still the case for VyOS version 1.5-rolling-202404030018

**Comment by robertcheramy** [comment]
Closed by PR #3112
